web/apps/apps.go

```diff
@@ -29,17 +30,59 @@ import (
 	"github.com/cozy/cozy-stack/pkg/couchdb"
 	"github.com/cozy/cozy-stack/pkg/jsonapi"
 	"github.com/cozy/cozy-stack/pkg/limits"
+	"github.com/cozy/cozy-stack/pkg/logger"
 	"github.com/cozy/cozy-stack/pkg/registry"
 	"github.com/cozy/cozy-stack/web/jobs"
 	"github.com/cozy/cozy-stack/web/middlewares"
 	"github.com/labstack/echo/v4"
+	"github.com/sirupsen/logrus"
 )
 
 // JSMimeType is the content-type for javascript
 const JSMimeType = "application/javascript"
 
 const typeTextEventStream = "text/event-stream"
 
+type AppLog struct {
+	Time  time.Time    "json:timestamp,string"
+	Level logrus.Level "json:level,string"
```

Comment:
I would prefer to declare `Level` as a `string` to avoid the `UnmarshalJSON` method, and just doing the `ParseLevel` later.